### ISSUE TITLE
Optimize package size

### ISSUE THREAD

**Issue by arjan-kitchenstories** [mujahidfa/vue-boring-avatars#3]
First of all, thank you for this port!

We would consider using the package if the bundle was smaller. Bundlephobia says it is 16KB large, while the underlying React library is around 3KB large. 

Is there a way to optimize the output? 16KB for creating customized avatars unfortunately would be a dealbreaker in my case.

Thanks again

**Comment by mujahidfa** [comment]
HI @arjan-kitchenstories! Upon inspection, it seems like the 16kb size you're referring to on Bundlephobia is **including** Vue's bundle size (which is a dependency for `vue-boring-avatars`, as it should). If you go to the bottom of the [page](https://bundlephobia.com/package/vue-boring-avatars@1.0.0) and hover over `(self)`, you can see that the actual size is **3.41kb unzipped**, which is a lot smaller than the React version at **14.9kb unzipped**.

Vue version at **3.41kb unzipped**:
![image](https://user-images.githubusercontent.com/17759705/133749156-7dd0185a-6b1f-44d7-acee-3e11ee8db93c.png)

vs

React version at **14.9kb unzipped**:
![image](https://user-images.githubusercontent.com/17759705/133750284-3989f11c-ad52-4f8d-86de-6c42ad4b2564.png)


The size that you see on the [React version's Bundlephobia](https://bundlephobia.com/package/boring-avatars@1.5.8) appears smaller because the author mistakenly omits React as a dependency when React should've been included.

![image](https://user-images.githubusercontent.com/17759705/133749709-c60d5e83-43d3-4129-9d57-ba0afe84d199.png)

TLDR; the Vue version's actual size are actually wayyy smaller than what you see on Bundlephobia since it includes Vue as a dependency.

Hope this addresses your concern!

**Comment by arjan-kitchenstories** [comment]
@mujahidfa Thank you for the swift response, thank you for clarifying - I should have seen that. Makes sense, and great news about the actual size!
